sortfilterproxymodel.qbs

```diff
@@ -46,6 +47,7 @@ Product
             return defines
         }
 
+        cpp.linkerFlags: "--no-as-needed"
```

Comment:
@crozeg Ok, interesting. I'll have to look at this more closely whenever I have the time, which probably won't be today...